### DISCUSSION TITLE
Proposal: Abandon ALL CAPS in future code Contributions

### DISCUSSION THREAD

**Discussion by NeverGray** [Open-SES/OpenSES#23] [Polls]
New OpenSES code contributions in Fortran should abandon All CAPS as appropriate. Lowercase letters are the current programming convention; see examples at https://www.fortran90.org/src/best-practices.html.

Also, using lowercase letters makes updates to 4.1 easier to spot. OpenSES's BSD 3-Clause License only applies to updates to SES 4.1. (One company has a misconception that OpenSES's license includes the original SES 4.1 code). Changing the letter convention helps people distinguish OpenSES code vs. SES 4.1 code.

This poll is related to pull request #22.

**Comment by TomODwyer-SES** [comment]
I'm against setting conventions like this.  The existing code is already entirely (or close to it) in upper case, why not follow that convention?  That convention is based on the hardware available when the program was written but has carried forward over time.

**Comment by NeverGray** [comment]
Chermac, Tom, and I concluded that the proposed new code should follow modern standards when practical. An example of these standards is provided on this website: https://www.fortran90.org/src/best-practices.html. In summary, the new code can abandon ALL CAPS.

I apologize for the delayed response. Chermac, Tom, and I met a few weeks ago to discuss this issue, but I only got around to it now.
